cypress/support/pages/cart.pageObject.js

```
@@ -0,0 +1,21 @@
+import PageObject from '../PageObject';
+
+class CartPageObject extends PageObject {
+  url = '/cart.html';
+
+  get cartItemNames() {
+    return cy.get('tr.success td:nth-child(2)');
+  }
+
+  verifyProductInCart(productName) {
+    this.cartItemNames
+      .should('contain.text', productName);
```

`verifyProductInCart(productName)` simply asserts `should('contain.text', productName)` on the collection. That's OK, but to reduce flakiness you could: 1) ensure you wait for the cart table to be visible before asserting (e.g. `cy.get('#tbodyid').should('be.visible')`), or 2) use `.contains()` to find an exact match row and assert existence so partial matches don't hide failures. The task requires asserting the product is present in the cart, so make the assertion explicit and robust.